README.ROCm.md

```diff
@@ -20,13 +20,3 @@ For details on installation and usage, see these links:
 * [Overview of ROCm port](rocm_docs/rocm-port-overview.md)
 * [List of supported operators on ROCm](rocm_docs/core_kernels.md)
 
-
-## Known Issues / Workarounds
-
-### tensorflow/benchmarks workaround for TF v1.8
-Since our current port of TF supports v1.8, we can't use some of the newest commits in `tensorflow/benchmarks`. RCCL, a ROCm version of NCCL, is under implementation. Therefore we have to drop back to an earlier point in the commit history, and disable NCCL.
```

Comment:
is this not true anymore?

Reply:
Right, if use the following branch of TF benchmarks:
https://github.com/tensorflow/benchmarks/tree/cnn_tf_v1.10_compatible